app/components/card/Card.js

```diff
@@ -27,7 +27,7 @@ export default class Card extends Component {
 
   render() {
     const placeholder =
-      '../../images/posterholder.png';
+      'images/posterholder.png';
```

Comment:
Sorry for the late answer, to package the app on linux 'icnsutils' and 'graphicsmagick' packages needs to be installed via

`sudo apt install graphicsmagick icnsutils`

I was getting the respective `ENOENT` errors.

Yes, it works, the image path is resolved without problems.